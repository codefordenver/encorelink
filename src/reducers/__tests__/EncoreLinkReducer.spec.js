import encoreLinkReducer from '../rootReducer';

describe('encoreLinkReducer', () => {
  it('has the expected initial state', () => {
    expect(encoreLinkReducer(undefined, {})).toMatchSnapshot();
  });
});
