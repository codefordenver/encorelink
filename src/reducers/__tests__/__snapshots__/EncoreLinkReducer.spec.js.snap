// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`encoreLinkReducer has the expected initial state 1`] = `
Object {
  "authData": Object {
    "isFetching": false,
    "isLoggedIn": false,
    "user": Object {},
    "userId": null,
    "userToken": null,
  },
  "form": Object {},
  "miscState": Object {
    "appInitiallized": false,
  },
  "models": Object {
    "allUrlData": Object {},
    "normalizedModels": Object {},
  },
}
`;
